src/server/services/buurt/datasets.ts

```diff
@@ -729,6 +730,84 @@ function transformParkeerzoneCoords(
   return features;
 }
 
+function transformSportparkResponse(
+  datasetId: DatasetId,
+  config: DatasetConfig,
+  responseData: any
+) {
+  const features = transformGenericApiListResponse(
+    datasetId,
+    config,
+    responseData,
+    'park'
+  );
+
+  return features;
+}
+
+function transformSportaanbiederResponse(
+  datasetId: DatasetId,
+  config: DatasetConfig,
+  responseData: any
+) {
+  const features = transformGenericApiListResponse(
+    datasetId,
+    config,
+    responseData,
+    'aanbieder'
+  );
+
+  return features;
+}
+
+function transformSporthalResponse(
+  datasetId: DatasetId,
+  config: DatasetConfig,
+  responseData: any
+) {
+  const features = transformGenericApiListResponse(
+    datasetId,
+    config,
+    responseData,
+    'hal'
+  );
+
+  return features;
+}
+
+function transformSportveldResponse(
+  datasetId: DatasetId,
+  config: DatasetConfig,
+  responseData: any
+) {
+  const features = transformGenericApiListResponse(
+    datasetId,
+    config,
+    responseData,
+    'veld'
+  );
+
+  // simplify soortOndergrond for types
+  const types = ['Kunstgras', 'Kunstrubber', 'Kushion Court', 'Top Clay'];
+
+  for (const feature of features) {
+    for (const type of types) {
+      if (feature.properties.soortOndergrond?.startsWith(type)) {
+        feature.properties.soortOndergrond = type;
+        break;
+      }
+    }
+  }
+
+  features.forEach((feature) => {
```

Comment:
ik kreeg dit niet voor elkaar met de valueConfig in myarea-datasets, dan kan geloof ik ook? Maar nu is het ook mooi gegroepeerd